docs/continuous-integration-and-deployment.md

```diff
@@ -1,30 +0,0 @@
-# Rally Web Platform - Continuous Integration and Deployment
```

Comment:
Did you mean to remove this? It's not super informative yet but I intend to expand on it :)

Reply:
No, this was an accident!